src/hammer-vlsi/synthesis/yosys/__init__.py

```diff
@@ -2,20 +2,34 @@
 # -*- coding: utf-8 -*-
 #
 # Yosys synthesis plugin for Hammer (as part of OpenROAD-flow installation).
+# mostly adoped from OpenLANE scripts found here: https://github.com/The-OpenROAD-Project/OpenLane/tree/master/scripts
```

Comment:
Looking at the scripts: I see some flags related to synthesis strategies and adder topologies. Do you intend to add these as keys?